channel/adjudicator.go

```diff
@@ -25,7 +25,7 @@ import (
 )
 
 //go:generate mockery --name AdjudicatorSubscription --output ../watcher/internal/mocks
-//go:generate mockery --name RegisterSubscriber --output ../watcher/internal/mocks
+//go:generate mockery --name WatcherAdjudicator --output ../watcher/internal/mocks
```

Comment:
@manoranjith introduced them for testing the local watcher. I need to adopt the interface for the watcher, if we were implementing watching of multi-ledger app channels. However, after @RmbRT comments I decided to focus on non-app channels first and reverted many of the mock changes. However, the mocks were still generated incorrectly, because the package and directory needs to be named `mocks` (instead of `mock`).